### ISSUE TITLE
Add Arm support

### ISSUE THREAD

**Issue by ruudboon** [hemtjanst/tradfri-mqtt#3]
Would it possible to add an arm image to the docker hub so we can run it on for example a Raspberry Pi?

**Comment by ruudboon** [comment]
I've managed to build the docker image on my Raspberry PI. Looks like it's connecting but then receives an empty payload. See error below.

```
In observe 15005 response ():
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Observer.onUpdate (/app/lib/observer.js:186:32)
    at PendingRequest.callback (/app/lib/observer.js:227:39)
    at Function.onMessage (/app/node_modules/node-coap-client/build/CoapClient.js:511:33)
    at SocketWrapper.emit (events.js:159:13)
    at SocketWrapper.socket.on (/app/node_modules/node-coap-client/build/lib/SocketWrapper.js:12:18)
    at Socket.emit (events.js:159:13)
    at Socket.udp_onMessage (/app/node_modules/node-dtls-client/build/dtls.js:204:34)
    at Socket.emit (events.js:159:13)
    at UDP.onMessage [as onmessage] (dgram.js:658:8)
```
When I console log the incoming message I see that it's receiving something but the buffer it receive is empty.

```
 Observer.prototype.onUpdate = function (url, r) {
        var payload = r.payload.toString();
        console.log(r, r.payload.length); 
```
Output
```
{ code: MessageCode { major: 4, minor: 1 },
  format: null,
  payload: <Buffer > } 0
```

**Comment by bonan** [comment]
What's the output if you start it with environment variable `DEBUG=tradfri*` or `DEBUG=*` enabled?

**Comment by bonan** [comment]
Ah, this is a known issue that I haven't had time to deal with yet since the gateway update that added requirement of authentication tokens.

The simple fix is to restart the container, it should start working after that.

You do want to mount a volume in `/data` though, so tradfri-mqtt can save it's `auth.json` there

**Comment by bonan** [comment]
The authentication issue is fixed in 21aa91f

I've also published an armv7 (RPi3+) on docker hub (`bonan/tradfri-mqtt:arm7`)
